Source/SwiftLintFramework/Rules/ValidDocsRule.swift

```diff
@@ -51,7 +51,9 @@ func declarationReturns(declaration: String, kind: SwiftDeclarationKind? = nil)
     guard let outsideBracesMatch = matchOutsideBraces(declaration) else {
         return false
     }
-
+    if declaration.containsString("init(") {
```

Comment:
This is dangerous, Consider a declaration of sorts:
`actionPerformFinit()` 
This would still match this case incorrectly. 

I think something like this would be better for this "Simple" style solution but I'm still not totally sure if checking for a string match is the best solution regardless. I think it could work for now if @jpsim thinks this is proper. 

```swift
guard !declaration.hasPrefix("init(") else {
    return true
}
```